src/components/ui/ReviewsCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import tyson from '../../assets/images/tyson.svg';
+import stars from '../../assets/images/star.svg';
+
+type ReviewsProps = {
+  image?: string;
+  name?: string;
+  rating?: number;
+  date?: string;
+  message?: string;
+};
+function ReviewsCard({
+  image, name, rating, date, message,
+}: ReviewsProps) {
+  return (
+    <div className="bg-white md:w-[700px] md:h-60 rounded-lg md:mt-2 drop-shadow-lg w-[300px] h-auto">
+      <div className="flex flex-row ml-5 pt-4">
+        <img className="w-11" src={image} alt="Profile" />
+        <h3 className="text-dark text-base font-semibold ml-5 pt-2">{name}</h3>
+      </div>
+      <div className="flex flex-row items-center ml-5 mt-3">
+        {/* Add stars component here */}
+        <img className="w-20" src={stars} alt="Ratings" />
```

Comment:
```suggestion
```
use star icons instead of a picture. might be a bit challenging but will teach you a lot.

Reply:
Yes, it's pretty nice